src/scripts/main.js

```diff
@@ -1,7 +1,21 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+  document.body.append(element);
+  element.classList.add('notification', type);
+
+  element.insertAdjacentHTML('beforeend', `
+  <h2 class='title'>${title}</h2>
+  <p>${description}</p>
```

Comment:
Let's improve  the readability 
```suggestion
    <h2 class='title'>${title}</h2>
    <p>${description}</p>
```

Reply:
Done, thank you!